installer-config.txt

```diff
@@ -8,7 +8,7 @@ userpw=raspberry
 boot_volume_label=openHABian
 disable_root=1
 user_is_admin=1
-username=pi
+username=admin
```

Comment:
Also, "admin" is the most targeted username for brute-force attempts, much more so than "Pi".